tests/scripts/project/hooks.cli

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env clitest --v0
+
+$ mktemp -d
+%SET WORK_DIR
+*
+
+$ cp -R ../../proj $WORK_DIR
+*
+
+$ gel instance destroy -I inst2 --force
+%EXIT any
+*
+
+# Create instance
+$ gel instance create inst2 default-branch-name --non-interactive
+*
+
+# Initialize project
+$ cd $WORK_DIR/proj/project3 && gel project init --link --server-instance=inst2 --non-interactive
+*
+! hook project.init.after: true
+*
+! hook migration.apply.before: true
+*
+! hook schema.update.before: true
+*
+! hook migration.apply.after: true
+*
+! hook schema.update.after: true
+*
```

Comment:
I think this is not testing that there were *no other hooks* ran in-between two hooks, right?

Reply:
True - I can reject other types of hook updates. I'll tweak the test.